### ISSUE TITLE
compilation error - cannot access CharScannerParser

### ISSUE THREAD

**Issue by hohwille** [m-m-m/entity#1]
Current CI build failed with this error:
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.9.0:compile (default-compile) on project mmm-entity-db: Compilation failure: Compilation failure: 
Error:  /home/runner/work/entity/entity/db/src/main/java/io/github/mmm/entity/bean/db/statement/AliasMap.java:[23,7] error: cannot access CharScannerParser
Error:    class file for io.github.mmm.scanner.CharScannerParser not found
```

Code looks perfectly fine in Eclipse and no errors.
Bug in Java compiler or module-system?

The class that he "cannot access" is obviously there:
https://github.com/m-m-m/scanner/blob/d7dee8a92af3ddf1fd335c175b6449dddd6f4096/core/src/main/java/io/github/mmm/scanner/CharScannerParser.java

Also the according module is required:
https://github.com/m-m-m/entity/blob/master/db/src/main/java/module-info.java#L13
https://github.com/m-m-m/entity/blob/master/bean/src/main/java/module-info.java#L15
https://github.com/m-m-m/bean/blob/master/factory/src/main/java/module-info.java#L16
https://github.com/m-m-m/bean/blob/master/core/src/main/java/module-info.java#L74
https://github.com/m-m-m/property/blob/master/builder/src/main/java/module-info.java#L13
https://github.com/m-m-m/validation/blob/master/main/src/main/java/module-info.java#L12
https://github.com/m-m-m/nls/blob/master/core/src/main/java/module-info.java#L130

**Comment by hohwille** [comment]
https://stackoverflow.com/questions/74635027/java-compiler-error-cannot-access-class-file-for-not-found
https://issues.apache.org/jira/browse/MCOMPILER-481

**Comment by hohwille** [comment]
I fixed it by downgrading compiler plugin. Proper fix will be to wait until 3.10.0 is released and update parent to it, then update parent version everywhere and remove the workaround.
